client/pages/sections/granted/project-summary.js

```diff
@@ -126,14 +122,14 @@ const ProjectSummary = ({
         <p className="legal">You are authorised to undertake this programme of scientific procedures at the following places:</p>
         <div className="granted-section">
           <h3>Primary establishment</h3>
-          <p>{ name }</p>
+          <p>{ establishment.name }</p>
           <dl className="inline">
             <dt>Establishment licence number: </dt>
-            <dd>{ licenceNumber }</dd>
+            <dd>{ establishment.licenceNumber }</dd>
             <dt>Establishment licence holder: </dt>
-            <dd>{ licenceHolder ? `${licenceHolder.firstName} ${licenceHolder.lastName}` : '-' }</dd>
```

Comment:
`licenceHolder` was confusing so I've removed the destructuring for establishment props.

Reply:
Can we use `licenceHolder` for the project licence holder then? `versionHolder` is _nasty_

Reply:
Or even better, can we structure this in the API so there's no UI change required at all?